src/olympia/migrations/954-add-file-is-restart-required.sql

```diff
@@ -0,0 +1,5 @@
+-- is_restart_required replaces the old no_restart field. It's the exact opposite.
+ALTER TABLE `files`
+    ADD COLUMN `is_restart_required` bool NOT NULL,
+    MODIFY `no_restart` bool;  -- Allow nulls to prepare for column removal next push.
+UPDATE `files` SET `is_restart_required` = NOT `no_restart`;
```

Comment:
Yep. I'm reasonably confident: I don't know about prod, but dev and stage have less than 170k rows each, and going through the table (`SELECT * FROM files ORDER BY RAND() LIMIT 10`) is pretty quick.